src/services/splitters.rs

```diff
@@ -230,6 +230,7 @@ pub fn split_slots_into_1h_slots(slots: Vec<Slot>) -> Vec<Slot> {
 ///         Task `Sleep` 00 - 06 (next day)
 /// ```
 pub fn split_crossed_tasks(tasks: &mut Vec<Task>) {
```

Comment:
@moaz-mokhtar 
Anything referring to `Task` should not be in the `placer` service.

Reply:
I just collected all functions that do some split or similar functionalities in a one module named `splitters`

Reply:
@moaz-mokhtar That is not a good idea. Just because it does something similar doesn't mean it should be grouped together.